controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1516,10 +1513,160 @@ bool master_thread::handle_intel_slave_join(Socket *sd, ieee1905_1::CmduMessageR
     return true;
 }
 
-bool master_thread::handle_non_intel_slave_join(Socket *sd,
-                                                std::shared_ptr<ieee1905_1::tlvWscM1> tlvwscM1,
-                                                std::string bridge_mac, std::string radio_mac,
-                                                ieee1905_1::CmduMessageTx &cmdu_tx)
+static inline std::vector<uint8_t> operating_class_to_channel_set(uint8_t operating_class)
+{
+    switch (operating_class) {
```

Comment:
Please add a comment (either here or in the doxygen header, but probably here) that explains where you get this table.